hibernate-reactive-core/src/test/java/org/hibernate/reactive/SubselectFetchTest.java

```diff
@@ -80,6 +82,11 @@ public void testQuery(VertxTestContext context) {
 							Node n2 = list.get( 1 );
 							assertFalse( isInitialized( n1.getElements() ), "'n1.elements' should not be initialized" );
 							assertFalse( isInitialized( n2.getElements() ), "'n2.elements' should not be initialized" );
+
+							// .size() triggers loading of elements
+							assertTrue( n1.getElements().size() == 3 );
+							assertTrue( n2.getElements().size() == 2 );
+
```

Comment:
Wait, we don't want to do this.

It's the call to `s.fetch( n1.getElements() )` that is supposed to trigger initialization of _both_ collections.

Also, you can't trigger loading by just calling `size()` in reactive.

Reply:
I should have worded it that `.size()` triggers initialization.

Does ORM now initialize both collections or only reactive code? (i.e. looking for a better example than `SubselectFetchTest`)

Reply:
The best thing is to read my explanation of subselect fetching in the new Guide to Hibernate 6, so you understand what it is we're trying to test here. (On phone right now so I don't have a link.)

But the idea is: the whole point of subselect fetching is that access to one collection triggers fetching of all of them.

Reply:
The link is https://docs.jboss.org/hibernate/orm/6.3/introduction/html_single/Hibernate_Introduction.html#batch-subselect-fetch.

By the way, to my eyes, the test looks correct as it was written, and shouldn't be changed.

Reply:
thx.  ... the changes I pushed actually worked locally, but I should have restarted workspace. It's still failing ...  as it should now.